src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -59,10 +59,6 @@ public function getMarking($subject)
     {
         $marking = $this->markingStore->getMarking($subject);
 
-        if (!$marking instanceof Marking) {
```

Comment:
The violation should be fixed, not every case where it could/will cause a problem.